pyproject.toml

```diff
@@ -140,5 +152,18 @@ max-args = 6
     "E402", # module level import not at top of file
 ]
 
+[tool.ruff.lint.flake8-copyright]
+notice-rgx = """
+# This code is a Qiskit project.
+
+# \\(C\\) Copyright IBM 2024\\.
+
+# This code is licensed under the Apache License, Version 2\\.0\\. You may
+# obtain a copy of this license in the LICENSE\\.txt file in the root directory
+# of this source tree or at http\\:\\/\\/www\\.apache\\.org\\/licenses\\/LICENSE\\-2\\.0\\.
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals\\.
```

Comment:
This is NOT the same copyright we used in the other repos.
There, we ensured that this is a block comment (i.e. no empty lines but an empty comment line: `#`) and we wrote the following on the first line:
> This code is part of a Qiskit project.

Which makes more sense to me, personally.
I didn't fix that here to minimize the effect of this PR, but at least CI will check this now.

Reply:
As discussed offline, are this (and the python file headers) good to stay as-is?

Reply:
Yes, but adding this check is a good idea :+1: